src/views/v2/emails/template-parts/body/order/ticket-totals/header-row.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * Event Tickets Emails: Order Ticket Totals - Header Row
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/v2/emails/template-parts/body/order/ticket-totals/header-row.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link https://evnt.is/tickets-emails-tpl Help article for Tickets Emails template files.
+ *
+ * @version TBD
+ *
+ * @since TBD
+ *
+ * @var Tribe_Template  $this  Current template object.
+ * @var Module           $provider              [Global] The tickets provider instance.
+ * @var string           $provider_id           [Global] The tickets provider class name.
+ * @var \WP_Post         $order                 [Global] The order object.
+ * @var int              $order_id              [Global] The order ID.
+ * @var bool             $is_tec_active         [Global] Whether `The Events Calendar` is active or not.
+ */
+
+ // @todo @codingmusician @juanfra Replace hardcoded data with dynamic data.
+
+?>
+<tr class="tec-tickets__email-table-content-order-ticket-totals-header-row">
+	<td style="width: 80%" class="tec-tickets__email-table-content-order-ticket-totals-cell tec-tickets__email-table-content-order-align-left" align="left">Ticket</td>
+	<td class="tec-tickets__email-table-content-order-ticket-totals-cell tec-tickets__email-table-content-order-align-center" align="center">Qty</td>
+	<td class="tec-tickets__email-table-content-order-ticket-totals-cell tec-tickets__email-table-content-order-align-right" align="right">Price</td>
```

Comment:
These strings will not come from the dynamic data, we should use gettext functions for them.